### DISCUSSION TITLE
v0.35.0

### DISCUSSION THREAD

**Discussion by mrmonday** [libpnet/libpnet#691] [Announcements]
## What's Changed
* Update license field following SPDX 2.1 license expression standard by @frisoft in https://github.com/libpnet/libpnet/pull/633
* transport: Add option to set ECN on the TransportSender socket. by @hawkinsw in https://github.com/libpnet/libpnet/pull/685
* Fix failing tests by @Paul-weqe in https://github.com/libpnet/libpnet/pull/676
* remove the repetitive word by @cuishuang in https://github.com/libpnet/libpnet/pull/672
* Add apple tvos support by @lcruz99 in https://github.com/libpnet/libpnet/pull/652
* Adding vxlan to pnet_packet by @stevedoyle in https://github.com/libpnet/libpnet/pull/654
* Add ICMP Destination unreachable Next-hop MTU by @fabi321 in https://github.com/libpnet/libpnet/pull/662
* Update ARP example to also support IPv6 via NDP by @tgross35 in https://github.com/libpnet/libpnet/pull/642
* Ensure BPF read is 4-byte aligned by @frankplow in https://github.com/libpnet/libpnet/pull/655
* Expose the various values in the TcpOption structure for external program access by @rikonaka in https://github.com/libpnet/libpnet/pull/640
* Definition for ethernet flow control packets. by @AJMansfield in https://github.com/libpnet/libpnet/pull/649
* Expose set_ecn on unix only by @mrmonday in https://github.com/libpnet/libpnet/pull/689
* datalink(linux): add feature to pass the fd (socket) to ::channel() by @Martichou in https://github.com/libpnet/libpnet/pull/584
* Added DNS protocol support by @tomDev5 in https://github.com/libpnet/libpnet/pull/678
* linux: use poll api instead of select inorder to support fd > 1024. Fixes #612 and #639 by @nemosupremo in https://github.com/libpnet/libpnet/pull/681

## New Contributors
* @frisoft made their first contribution in https://github.com/libpnet/libpnet/pull/633
* @hawkinsw made their first contribution in https://github.com/libpnet/libpnet/pull/685
* @Paul-weqe made their first contribution in https://github.com/libpnet/libpnet/pull/676
* @cuishuang made their first contribution in https://github.com/libpnet/libpnet/pull/672
* @lcruz99 made their first contribution in https://github.com/libpnet/libpnet/pull/652
* @stevedoyle made their first contribution in https://github.com/libpnet/libpnet/pull/654
* @fabi321 made their first contribution in https://github.com/libpnet/libpnet/pull/662
* @tgross35 made their first contribution in https://github.com/libpnet/libpnet/pull/642
* @frankplow made their first contribution in https://github.com/libpnet/libpnet/pull/655
* @AJMansfield made their first contribution in https://github.com/libpnet/libpnet/pull/649
* @tomDev5 made their first contribution in https://github.com/libpnet/libpnet/pull/678
* @nemosupremo made their first contribution in https://github.com/libpnet/libpnet/pull/681

**Full Changelog**: https://github.com/libpnet/libpnet/compare/v0.34.0...v0.35.0

<hr /><em>This discussion was created from the release <a href='https://github.com/libpnet/libpnet/releases/tag/v0.35.0'>v0.35.0</a>.</em>

**Comment by Akmot9** [comment]
Hello everyone,

Bravo on the release of v0.35.0! 🎉 The updates and enhancements are impressive. I actively use this library, and I appreciate the hard work and contributions from the maintainers and new contributors. The addition of features DNS protocol support are particularly exciting. Keep up the fantastic work!

Best,
Cyprien
